__tests__/app.test.js

```diff
@@ -36,3 +36,32 @@ describe("'Get /api", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id", () => {
+  test("status 200, responds with the correct article for a valida ID", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then((res) => {
+        expect(res.body.article).toHaveProperty("article_id", 1);
+      });
+  });
```

Comment:
You should be asserting for every property we expect not just article_id. If I wanted to send a request for a specific article I should expect to receive: 
- author
- title
- article_id (which you already have)
- body
- article_img_url
- topic
- created_at
- and votes

I would recommending looking into either the .objectContaining(...) or .toMatchObject(...) matchers for jest as these are very flexible.